src/views/SensorProvisioningGuide.jsx

```diff
@@ -194,8 +194,8 @@ function SensorProvisioningGuide() {
             <div style={styles.stepText}>
               <ol start="6">
                 <li style={styles.stepTitle}>
-                  Obtain an IM21 Door Sensor and retrieve it&apos;s BLE ID (shown in image) and use the{' '}
-                  <a href="/door-sensor-pairing">Door Sensor Pairing Tool</a> to pair the IM21 to the Boron.
+                  Obtain an IM21 Door Sensor and retrieve it&apos;s BLE ID (shown in image) and use the Particle console to pair the IM21 to the
```

Comment:
What do you think about updating the `{Step6}` image to remove the screenshot of the PA Door Pairing screen?
It looks like this image will need to be updated as part of your IM24 changes (https://app.clickup.com/t/860ravv7d) too. So maybe it can all be done in one go?